internal/commands/push/command.go

```diff
@@ -39,7 +39,7 @@ func (cmd *Command) Flags() []flags.Flag {
 		flags.StringFlag{
 			Value: &cmd.inputs.LocalPath,
 			Meta: flags.Meta{
-				Name: flagLocalPath,
+				Name: flags.FlagLocalPath,
```

Comment:
probably don't need to use `const` here

the decision was intentional to not keep a global flag name `const` list.  basically, we felt that letting these commands be as un-coupled as possible to other commands was preferable.  the trade-off here is there are a few patterns and practices we loosely adhere to (like remember to use `"local"` and/or `"remote"` when talking about apps).